tests/requirements.in

```diff
@@ -1,11 +1,7 @@
-flexmock
-freezegun
-pytest >= 3 # Needed for compatibility with celery 4.x fixtures
-pytest-flask
-pytest-cov
-requests
-jsonschema
-pylint
+pytest
 semantic_version
-pydantic
```

Comment:
You can include development requirement file first and than add test dependencies.